frontend/src/components/AddIngredients.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components/macro";
+import { useState } from "react";
+
+export default function AddIngredients({ onAddClick }) {
```

Comment:
I am not quite sure why you are passing props onAddClick here

Reply:
Function will be added soon.